src/find_datarefs_in_files.cpp

```diff
@@ -136,7 +136,30 @@ std::vector<std::string> getDatarefsFromAircraft(const std::string & acf_path) {
 		}
 
 		if(boost::filesystem::is_regular_file(path)) {
-			if(".obj" == path.extension() || ".acf" == path.extension()) {
+			if(".obj" == path.extension() || ".acf" == path.extension() || ".lua" == path.extension()) {
+				std::vector<std::string> refs = getDatarefsFromFile(path.string());
+				all_refs.insert(all_refs.begin(), refs.begin(), refs.end());
+			}
+		}
+	}
+
+	//lua files. Recurse over Custom Avionics directory structure
+	paths.push_back(aircraft_dir / "Custom Avionics");
+
+	while(false == paths.empty()) {
```

Comment:
Bill this loop duplicates some of the functionality of the while() loop in the code below. I've tried to basically copy this functionality into 636dfbd0cc292e568b7cc83b45b053011e04c7ff, can you verify that commit finds the datarefs that you were missing?